lib/battle_boats/random_fleet_deployer.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "board"
+require_relative "coordinate"
+
+module BattleBoats
```

Comment:
Can you clarify why this class is being namespaced by the BattleBoats module

Reply:
I may be incorrect, but it's my understanding that all classes inside of Ruby gems are conventionally namespaced within a module. This is in order to avoid name collisions between the gem and any application code that uses it.